_quarto.yml

```diff
@@ -66,7 +66,9 @@ website:
           - cohorts/NMFS-fall2021.md
           - cohorts/NEFSC-spr2020.md            
       - href: data-academy.qmd
-        text: Data Academy
+        text: Data Academy           
+      - href: coworking-opportunities.qmd
+        text: Coworking Opportunities
```

Comment:
I wonder if  "Live Coworking" "Coworking - Live with others" or "Coworking Times" or something would be punchier? On the rendered navbar it gets a bit lost, and I wonder if that's because of the word choice & length ("Opportunities")

Let's also move this to the end. Let's do the sidebar order (going from static to dynamic):

- presentations
- resources
- blog
- contact
- live coworking